src/index.css

```diff
@@ -1,9 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
+  margin: 20px;
   text-align: center;
+  align-items: center;
+  margin-left: 12rem;
+  background-color: rgb(204, 209, 211);
+}
+
+.navigation-bar {
+  display: flex;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  background-color: #fff;
+  border-bottom: 1px solid gray;
+  margin-left: 4rem;
+  margin-top: 1.5rem;
+}
+
```

Comment:
- [ ] As the previous reviewer said, you should follow the [Bookstore template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). It would be better if you use the right color for the title and buttons. ♻️ 

![Color-bookstore](https://user-images.githubusercontent.com/79876959/142715310-95db9832-e772-4cd6-b17f-724bf71c4472.PNG)